pkg/controller/servicebindingrequest/retriever.go

```diff
@@ -26,6 +26,7 @@ type Retriever struct {
 	volumeKeys    []string                     // list of keys found
 	bindingPrefix string                       // prefix for variable names
 	cache         map[string]interface{}       // store visited paths
+	Annotation    bool                         // If true use CRD annotation otherwise fallback to CSV
```

Comment:
why is `Annotation` an exported field?

Reply:
Removed. e67ab21318272503bc6a3d356fd9a66282d19eea